docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3140,32 +3140,32 @@ Noise should be added in the order below.
       - Pseudopeople name
     * - "Borrowed" SSN
       - Borrow a social security number
+    * - Guardian-based duplication
+      - Duplicate with guardian
+    * - (Simple) Duplication
+      - (not yet implemented)
```

Comment:
```suggestion
```
Let's omit this noise type for now since it's unclear what it means. If we include something like this, it should probably happen around the same time as "simple omission," in particular, after "Do not respond." But we haven't decided how we want this to work, so let's not even mention it.